web/src/components/LogosList/LogosList.module.scss

```diff
@@ -0,0 +1,15 @@
+.img {
+  height: auto;
+  margin-top: 1rem !important;
+  width: 212px;
+}
+
+.span {
+  margin-bottom: 1rem !important;
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.div {
+  margin-bottom: 1rem !important;
+}
```

Comment:
Could we give styles.div, styles.span and styles.img more semantic names? Maybe styles.logoContainer, styles.logoHolder and styles.logo?